### PR TITLE
Fix write-index deletion in delete by percentage

### PR DESCRIPTION
**Pull request by btaani**
### Description
This PR fixes the accidental deletion of write-indices in case multiple index aliases belong to the same policy (e.g., `app-*` and `app-test-*`).

/assign @periklis 

### Links
- Depending on PR(s): #848 and #865 
- JIRA: [LOG-2444](https://issues.redhat.com/browse/LOG-2444)

### CODE REVIEW

**Review by periklis — COMMENTED** [review]
Look good to me

File: internal/indexmanagement/scripts.go [hunk 1]
```diff
@@ -132,6 +132,35 @@ def updateWriteIndex(currentIndex, nextIndex, alias):
...
+def indexBelongsToAlias(index, alias):
+  original_stdout = sys.stdout
+  try:
+    es_client = getEsClient()
+    response = es_client.indices.get_alias(index=index)
+    writeIndexName = list(key for key, value in response[index]["aliases"].items() if '-write' in key)[0]
+    if alias+"-write" == writeIndexName:
+      return True
+    else:
+      return False
```
Comment (Red-GV) [inline internal/indexmanagement/scripts.go:144]: Small nitpick. 
```suggestion
    return alias+"-write" == writeIndexName
```

**Review by Red-GV — COMMENTED** [review]
Overall, looks good to me.

**Pull request merged**